yt_dlp/extractor/bannedvideo.py

```diff
@@ -0,0 +1,235 @@
+from __future__ import unicode_literals
+
+import json
+
+from .common import InfoExtractor
+
+from ..utils import (
+    try_get,
+    int_or_none,
+    url_or_none,
+    float_or_none,
+    unified_timestamp,
+    unified_strdate,
+)
+
+
+class BannedVideoIE(InfoExtractor):
+    _GRAPHQL_API = 'https://api.infowarsmedia.com/graphql'
+    _GRAPHQL_HEADERS = {
+        'Content-Type': 'application/json; charset=utf-8'
+    }
+    _GRAPHQL_GETVIDEO_QUERY = '''
+query GetVideo($id: String!) {
+    getVideo(id: $id) {
+        ...DisplayVideoFields
+        streamUrl
+        directUrl
+        unlisted
+        live
+        tags {
+            _id
+            name
+            __typename
+        }
+        sale {
+            _id
+            text
+            __typename
+        }
+        __typename
+    }
+}
+fragment DisplayVideoFields on Video {
+    _id
+    title
+    summary
+    playCount
+    largeImage
+    embedUrl
+    published
+    videoDuration
+    channel {
+        _id
+        title
+        avatar
+        __typename
+    }
+    createdAt
+    __typename
+}'''
+    _GRAPHQL_GETCOMMENTS_QUERY = '''
+query GetVideoComments($id: String!, $limit: Float, $offset: Float) {
+    getVideoComments(id: $id, limit: $limit, offset: $offset) {
+        ...VideoComment
+        replyCount
+        __typename
+    }
+}
+fragment VideoComment on Comment {
+    _id
+    content
+    liked
+    user {
+        _id
+        username
+        __typename
+    }
+    voteCount {
+        positive
+        __typename
+    }
+    linkedUser {
+        _id
+        username
+        __typename
+    }
+    createdAt
+    __typename
+}'''
+    _GRAPHQL_GETCOMMENTSREPLIES_QUERY = '''
+query GetCommentReplies($id: String!, $limit: Float, $offset: Float) {
+    getCommentReplies(id: $id, limit: $limit, offset: $offset) {
+        ...VideoComment
+        replyTo {
+            _id
+            __typename
+        }
+        __typename
+    }
+}
+fragment VideoComment on Comment {
```

Comment:
This `fragment` is the same as on line 69-89. Could be represented once and the concatenated to the comment queries.